sdk/tests/js/tests/gl-object-get-calls.js

```diff
@@ -964,6 +964,16 @@ if (contextVersion > 1) {
 	    return gl.getActiveUniforms(program, uniformIndices, pname);
         }
     );
+
+    debug("");
+    debug("Test getUniformBlockIndex");
+    var program = wtu.loadUniformBlockProgram(gl);
+    gl.linkProgram(program);
+    shouldBeTrue('gl.getProgramParameter(program, gl.LINK_STATUS)');
+    shouldBeNonNull('gl.getUniformBlockIndex(program, "Transform")');
+    var blockIndex = gl.getUniformBlockIndex(program, "Transform");
+    gl.uniformBlockBinding(program, blockIndex, 1);
+    shouldBe('gl.getUniformBlockIndex(program, "Transform")', '1');
```

Comment:
Zhenyao and Ken, you are correct. We should use getActiveUniformBlockParameter to check that it has been bound to a correct uniform buffer binding point. However, I advice @renchenglei to submit another patch to add the tests you suggested against getActiveUniformBlockParameter. For this one, let's keep it simple. 

I would like to add some invalid test cases against getUniformBlockIndex: 
1) test a uniform name (not uniform block name) by this API. 
2) test a non-existed uniform block name.
3) test a program without any uniform blocks at all (but may have uniforms). 
@renchenglei , could you add these test cases?